src/lib/fragments/Pileup.react.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
+import {propTypes, defaultProps} from '../components/Pileup.react';
+import pileup from 'pileup';
+
+import '../../../node_modules/pileup/style/pileup.css';
+
+/**
+ * The Pileup component is an genome visualization component
+ * developed by the the Hammerlab. It uses an
+ * example integration of pileup.js and React (https://www.npmjs.com/package/pileup).
+ */
+export default class Pileup extends Component {
+    constructor(props) {
+        super(props);
+        this.ref = React.createRef();
+        this.pileup = null;
+    }
+
+    parseTracks(reference, tracks) {
```

Comment:
I'll just echo Alex's comments from a similar PR here: Although it seems to work fine without it, we'll want to bind any methods here that aren't built-in React methods to `this` in the `constructor` to make sure it is called in the right context. Here's an example of how `circos` does it:
 https://github.com/plotly/dash-bio/blob/3010db8dea220daaf3ddaf4d22fa812022a8b4eb/src/lib/fragments/Circos.react.js#L17-L23